### PR TITLE
[zh] Sync concepts pages for garbage collect

### PR DESCRIPTION
**Pull request by chenxuc**
Sync web page for part of the items in umbrella issue:
[zh] Umbrella issue: pages out of sync in concepts section #26177

The pages included in this PR:
Garbage Collection
 content/zh/docs/concepts/workloads/controllers/garbage-collection.md

### CODE REVIEW

**Comment by chenxuc** [comment]
Hi QiMing,
Sorry for my progress is a bit slow due to my high workload before Spring holiday. 

/assign tengqm

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+with a reason of `OwnerRefInvalidNamespace` and an `involvedObject` of the invalid dependent is reported.
+You can check for that kind of Event by running
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`.
 -->
 {{< note >}}
-根据设计，kubernetes 不允许跨命名空间指定属主。这意味着：
-1）命名空间范围的附属只能指定同一的命名空间中的或者集群范围的属主。
-2）集群范围的附属只能指定集群范围的属主，不能指定命名空间范围的属主。
-{{< /note >}}
+根据设计，kubernetes 不允许跨命名空间指定属主。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:114]: 命名空间 -> 名字空间，下同。

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 2]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+
+命名空间的附属可以指定集群范围的或者命名空间范围的属主。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:116]:
```suggestion
名字空间范围的附属可以指定集群范围的或者名字空间范围的属主。
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 3]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+
+命名空间的属主**必须**和该附属处于相同的命名空间。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:118]:
```suggestion
名字空间范围的属主**必须**和该附属处于相同的名字空间。
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 4]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+如果命名空间的属主和附属不在相同的命名空间，那么该属主引用就会被认为是缺失的，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:119]: 不能译为”命名空间的属主“，会被理解为”owner of namespace"
应译为“名字空间范围的属主”

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 5]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+并且当附属的所有属主引用都缺失之后，该附属就会被删除。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:120]:
```suggestion
并且当附属的所有属主引用都被确认不再存在之后，该附属就会被删除。
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 6]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+
+集群范围的附属只能指定集群范围的属主。
+在 v1.20+ 版本，如果一个集群范围的附属指定了一个命名空间范围类型的属主，
+那么该附属就会被认为是拥有一个不可解析的属主引用，并且它不能够被垃圾回收。
+
+在 v1.20+ 版本，如果垃圾收集器检测到无效的跨命名空间的属主引用，
+或者一个集群范围的附属指定了一个命名空间范围类型的属主，
+那么它就会报告一个警告事件。该事件的原因是`OwnerRefInvalidNamespace`，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:128]:
```suggestion
那么它就会报告一个警告事件。该事件的原因是 `OwnerRefInvalidNamespace`，
```

注意中英文间留白。

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 7]
```diff
@@ -93,18 +93,45 @@ metadata:
...
+`involvedObject`属性是该无效的附属。你可以通过以下命令来获取该类型的事件：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:129]:
```suggestion
`involvedObject` 属性中包含无效的附属。你可以通过以下命令来获取该类型的事件：
```

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 8]
```diff
@@ -93,18 +93,45 @@ metadata:
...
 
+```shell
+kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace
+```
+{{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:134]: 请尽量逐段翻译，不要一次性翻译过长的段落，不方便 review

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 9]
```diff
@@ -93,18 +93,47 @@ metadata:
...
-根据设计，kubernetes 不允许跨命名空间指定属主。这意味着：
-1）命名空间范围的附属只能指定同一的命名空间中的或者集群范围的属主。
-2）集群范围的附属只能指定集群范围的属主，不能指定命名空间范围的属主。
-{{< /note >}}
+根据设计，kubernetes 不允许跨名字空间指定属主。
+
+名字空间范围的附属可以指定集群范围的或者名字空间范围的属主。
+
+名字空间范围的属主**必须**和该附属处于相同的名字空间。
+如果名字空间的属主和附属不在相同的名字空间，那么该属主引用就会被认为是缺失的，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:109]:
```suggestion
如果名字空间范围的属主和附属不在相同的名字空间，那么该属主引用就会被认为是缺失的，
```

**Comment by tengqm** [comment]
need to squash the commits to one

**Comment by chenxuc** [comment]
Hi QiMing,
Thanks a lot for the very detailed suggestions. I've updated the document for all comments.
Working on the document requires much more skills than I expected, and it's more exiting when finally done~

**Comment by chenxuc** [comment]
> need to squash the commits to one

Hi QiMing,
I used to squash commits during merge in company github. Do you mean I need to squash commit before merge? I haven't done so before. Is it something like https://gist.github.com/longtimeago/f7055aa4c3bba8a62197 ? 
Appreciate for the advice!

**Comment by tengqm** [comment]
@chenxuc That is what you need to do before this is merged.

**Comment by chenxuc** [comment]
> @chenxuc That is what you need to do before this is merged.

Thanks a lot! 
So excited to learn so many things in one single task. I should get familiar with the doc tasks through it, and speed up the future tasks.

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md (hunk 8 above)
Comment (ydFu) [reply content/zh/docs/concepts/workloads/controllers/garbage-collection.md:136]: @tengqm 
Thank you for your effort.

**Comment by ydFu** [comment]
Hint: running git rebase -I then git push -f.

**Comment by chenxuc** [comment]
@tengqm 
Hi QiMing, I've squashed the commit. Please let me know if anything else needed. Thanks

**Pull request merged**